crowdin_updater.py

```diff
@@ -122,13 +125,17 @@ def main():
     parser.add_argument('-f', dest='inputfile', help='Update file path')
     parser.add_argument('--un', dest='update_nuxeo', action='store_true', help='Update Nuxeo from Crowdin')
     parser.add_argument('-o', dest='outputdir', help='Output directory')
+    parser.add_argument('-p', dest='parentdir', help='Crowding parent folder')
     args = parser.parse_args()
 
     cu = CrowdinUpdater(args.project, format=args.format)
     if args.update_crowdin:
         cu.upload(args.inputfile)
     if args.update_nuxeo:
-        cu.download(args.outputdir)
+        if args.parentdir:
+            cu.download(args.outputdir, parentdir=args.parentdir)
+        else:
+            cu.download(args.outputdir)
```

Comment:
This could be replaced by

    cu.download(args.outputdir, parentdir=args.parentdir)

with

    def download(self, outputdir, build=True, package=None, parentdir=None):

and inside the `download` function

    if parentdir:
        ...